pkg/controller/pxc/status.go

```diff
@@ -235,13 +235,18 @@ func (r *ReconcilePerconaXtraDBCluster) appStatus(app api.StatefulApp, namespace
 	return status, nil
 }
 
-func (r *ReconcilePerconaXtraDBCluster) appHost(app api.StatefulApp, namespace string, podSpec *api.PodSpec) (string, error) {
+func (r *ReconcilePerconaXtraDBCluster) appHost(cr *api.PerconaXtraDBCluster, app api.StatefulApp, podSpec *api.PodSpec) (string, error) {
 	if podSpec.ServiceType != corev1.ServiceTypeLoadBalancer {
-		return app.Service() + "." + namespace, nil
+		return app.Service() + "." + cr.Namespace, nil
```

Comment:
@pooknull @egegunes I think we need to use` cr.Name + "-proxysql"` in this case too. We should not use unready service for ENDPOINT

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1420/commits/d48179f1a1e3e0a0fd6154cf76635906fe38c353